src/m365/commands/docs.spec.ts

```diff
@@ -0,0 +1,82 @@
+import Command from '../../Command';
+import commands from './commands';
+import * as assert from 'assert';
+import * as sinon from 'sinon';
+import { Logger } from '../../cli/Logger';
+import * as open from 'open';
+import { telemetry } from '../../telemetry';
+import { pid } from '../../utils/pid';
+import { session } from '../../utils/session';
+import { sinonUtil } from '../../utils/sinonUtil';
+import { Cli } from '../../cli/Cli';
+const packageJSON = require('../../../package.json');
+const command: Command = require('./docs');
+
+describe(commands.DOCS, () => {
+  let log: any[];
+  let logger: Logger;
+  let cli: Cli;
+  let loggerLogSpy: sinon.SinonSpy;
+  let getSettingWithDefaultValueStub: sinon.SinonStub;
+  let openStub: sinon.SinonStub;
+
+  before(() => {
+    sinon.stub(telemetry, 'trackEvent').callsFake(() => { });
+    sinon.stub(pid, 'getProcessName').callsFake(() => '');
+    sinon.stub(session, 'getId').callsFake(() => '');
+  });
+
+  beforeEach(() => {
+    log = [];
+    cli = Cli.getInstance();
+    logger = {
+      log: (msg: string) => {
+        log.push(msg);
+      },
+      logRaw: (msg: string) => {
+        log.push(msg);
+      },
+      logToStderr: (msg: string) => {
+        log.push(msg);
+      }
+    };
+    loggerLogSpy = sinon.spy(logger, 'log');
+    (command as any)._open = open;
```

Comment:
Can't we remove this line?

Reply:
This line ensures that a call to `command._open` is replaced with the stubbed function that resolves with no value. This is done in consistency with other commands, as creating a stub for the `open` package is not working.

Let me know if I can keep it as is?

Reply:
I see, in fact we could dump any random value in it in that case. But ok, let's leave it as is.